src/ceph-volume/plugin/zfs/setup.cfg

```diff
@@ -0,0 +1,9 @@
+[bdist_wheel]
+universal = 1
```

Comment:
this project is not distributed using pypi. i don't see under what circumstances we need to build a universal `whl` for it.